scripts/go-wrapper.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
Could you add `set -e` in here as well to stop execution upon a command failure?